src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -19,7 +20,7 @@ def create(data,
            max_iterations=0,
            verbose=True):
     """
-    Create a :class:`ObjectDetector` model.
+    Create a class: `OneShotObjectDetector` model.
```

Comment:
:class:`class_name` is formatting for API docs.

Reply:
Yeah just rebasing with master. Realized that. Will be pushing the updated code in a bit.